gcs-authenticator/mod.go

```diff
@@ -48,7 +55,17 @@ func main() {
 
 	logger := log.New(os.Stdout, "http: ", log.LstdFlags)
 
-	jsonKey, err := ioutil.ReadFile(os.Getenv("GCS_PRIVATE_PATH"))
+	gcs_key_path := os.Getenv(GCS_KEY_PATH)
+	if gcs_key_path == "" {
+		logger.Fatalf("please set %s", GCS_KEY_PATH)
+	}
+
+	gcs_bucket_name := os.Getenv(BUCKET_KEY_NAME)
+	if gcs_bucket_name == "" {
+		logger.Fatalf("please set %s", BUCKET_KEY_NAME)
+	}
+
+	jsonKey, err := ioutil.ReadFile(os.Getenv(GCS_KEY_PATH))
```

Comment:
why do you not use `gcs_key_path` variable declared above ?

Reply:
Nice spot!